test/GitcoinGovernor.t.sol

```diff
@@ -48,7 +48,7 @@ contract GitcoinGovernorDeployTest is GitcoinGovernorTestHelper {
     assertEq(governorBravo.proposalThreshold(), INITIAL_PROPOSAL_THRESHOLD);
     assertEq(governorBravo.quorum(_blockNumber), QUORUM);
     assertEq(governorBravo.timelock(), TIMELOCK);
-    assertEq(governorBravo.COUNTING_MODE(), "support=bravo&quorum=bravo&params=fractional");
+    assertEq(governorBravo.COUNTING_MODE(), "support=bravo&quorum=for,abstain&params=fractional");
```

Comment:
We need to confirm that this is okay. Tracking here https://github.com/gitcoinco/Alpha-Governor-Upgrade/issues/18